samples/start-app/python-fastapi/app/dependencies.py

```diff
@@ -17,6 +17,9 @@ def get_config(name):
 def get_session():
     return session
 
+def openid_authentication():
+    return get_config('OPENID_AUTHENTICATION') == 'true' or get_config('OPENID_AUTHENTICATION')  == '1' or get_config('OPENID_AUTHENTICATION') == True;
```

Comment:
Reading this https://www.freecodecamp.org/news/truthy-and-falsy-values-in-python/ I think you do not have to test each of these value. In every cases value should be read as trhuty

Reply:
When using `get_config('OPENID_AUTHENTICATION')` method, a string value is returned. However, it's important to note that when comparing the returned value with `'0'` or `'false'`, the comparison may return true even if the value is not actually equal to `0` or `false`

Reply:
ok I did not know that you where comparing a string :)